routes/auth.routes.js

```diff
@@ -0,0 +1,79 @@
+const { Router } = require("express");
+const router = new Router();
+
+const bcryptjs = require("bcryptjs");
+const User = require('../models/User.model');
+saltRounds = 10;
```

Comment:
When creating a variable try and use either let or const. When you declare a variable without anything behind, it changes it to "var" when the code runs